cypress/integration/bookshelf_spec.js

```diff
@@ -7,8 +7,8 @@ describe('BookshelfDisplay', () => {
   it('Should display the navbar', () => {
     cy.get('.nav').should('be.visible')
       .get('nav > h1').should('contain', 'BookWorm')
-      .get('nav > ul > li').eq(0).should('contain', 'Home')
-      .get('nav > ul > li').should('contain', 'Favorites')
+      .get('nav > ul > a').eq(0).should('contain', 'Home')
+      .get('nav > ul > a').should('contain', 'Favorites')
```

Comment:
Nice job keeping the code more dry by notating into child elements and chaining should with contain!